relentless/simulate/lammps.py

```diff
@@ -512,107 +506,87 @@ def to_commands(self, sim):
 
         return cmds
 
-class AddNPTIntegrator(LAMMPSOperation):
-    r"""NPT integration via Nos\'e-Hoover thermostat/barostat.
+class Thermostat:
```

Comment:
I just realized I am reading this in `simulate/lammps.py`. These thermostat and barostat objects are general (not specific to any specific engine), so you should go in `simulate/simulate.py` then imported into `simulate/__init__.py`. You can remove from here and HOOMD. We want the user to use them like:

```
relentless.simulate.NoseHooverThermostat(T=1.0, tau=0.1)
```